packages/account/src/Sections/Profile/LanguageSettings/settings-language.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```

Comment:
This component is available in `packages/core/src/App/Containers/SettingsModal/settings-language.jsx`, can we move this component to the `shared` package then use it in both `core` and `appstore` packages?